docs/en/operations/troubleshooting.md

```diff
@@ -17,6 +17,37 @@ title: Troubleshooting
 -   Check firewall settings.
 -   If you cannot access the repository for any reason, download packages as described in the [install guide](../getting-started/install.md) article and install them manually using the `sudo dpkg -i <packages>` command. You will also need the `tzdata` package.
 
+### You Cannot Update Deb Packages from ClickHouse Repository with Apt-get {#you-cannot-update-deb-packages-from-clickhouse-repository-with-apt-get}
+
+- The issue may be happened when the GPG key is changed.
+
+Please use the following scripts to resolve the issue:
+
+```bash
+sudo apt-key del "3A9E A119 3A97 B548 BE14  57D4 8919 F6BD 2B48 D754"
+sudo rm /etc/apt/sources.list.d/clickhouse.list
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 8919F6BD2B48D754
```

Comment:
It's completely redundant. Just repeating the command `sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 8919F6BD2B48D754` updates the key

Reply:
Just simplify the GPG key updating approach.